packages/react-admin/package.json

```diff
@@ -36,14 +36,12 @@
         "@emotion/styled": "^11.3.0",
         "@mui/icons-material": "^5.0.1",
         "@mui/material": "^5.0.2",
-        "final-form": "^4.20.2",
+        "final-form": "^4.20.4",
```

Comment:
seems unrelated to this PR

Reply:
That's just a line that moved, most probably automatically